hapi/hedera-protobufs/services/state/roster/ledger_id.proto

```diff
@@ -61,8 +67,10 @@ option java_multiple_files = true;
 message LedgerId {
 
   /**
-   * A public key.<br/>
-   * This key both identifies the ledger and can be used to verify ledger signatures.
+   * A public key.
+   * <p>
+   * This key both identifies the ledger and can be used to verify ledger
+   * signatures.
```

Comment:
This was correct before this change.
Several other examples below with the same issue.
```suggestion
   * A public key.<br/>
   * This key both identifies the ledger and can be used to verify ledger
   * signatures.
```